script.js

```diff
@@ -0,0 +1,74 @@
+const bookList = document.querySelector('.book-lists');
+const addBtn = document.getElementById('submit');
+const bookTitle = document.getElementById('book-title');
+const bookAuthor = document.getElementById('book-author');
+const key = 'BOOKS_LIST';
+
+let BookObj = [];
+
+class Books {
+  constructor() {
+    if (JSON.parse(localStorage.getItem(key)) != null) {
+      this.BookObj = JSON.parse(localStorage.getItem(key));
+    } else {
+      this.BookObj = [];
+    }
+  }
+
+  add(book, author) {
+    this.BookObj.push({
+      id: this.BookObj.length,
+      title: book.value,
+      author: author.value,
+    });
+    localStorage.setItem('BOOKS_LIST', JSON.stringify(this.BookObj));
+  }
+
+  remove(element) {
+    const { id } = element.dataset;
+    element.parentElement.remove();
+    this.BookObj.splice(this.BookObj.findIndex((item) => item.id === parseInt(id, 10)), 1);
+    localStorage.setItem('BOOKS_LIST', JSON.stringify(this.BookObj));
+  }
```

Comment:
- [ ] Good job working on this but at the moment when I add books and start removing them from the top, it removes the wrong book from the storage. Please kindly ensure that removing an item from the DOM actually removes the right item in the storage.